Podcast/EndpointRequest.swift

```diff
@@ -101,7 +101,7 @@ class EndpointRequest: Operation {
         return baseURLString + path
     }
     
-    func parameters() -> [String:Any] {
+    func parameters() -> [String:Any]? {
```

Comment:
Needs to be changed because if there are no query parameters it should return nil ... rn it's returning [:] which gave me a bug in one of the endpoint requests